tests/Model/CodecCursorFunctionalTest.php

```diff
@@ -55,12 +55,15 @@ function (...$args) use (&$previousErrorHandler, &$deprecations) {
                 E_USER_DEPRECATED | E_DEPRECATED,
             );
 
-            $cursorId = $codecCursor->getId();
+            $cursorId = $codecCursor->getId(true);
```

Comment:
With this change the error handler should no longer be necessary, and there should not be any deprecations to assert against.

Reply:
Yes, I released this should be reverted if we want to validate the exception.